### DISCUSSION TITLE
Inspiration: Notenik

### DISCUSSION THREAD

**Discussion by minthemiddle** [notaapp/talk#170] [Random]
Yet another inspiring app, Notenik.
The idea: Create a structure with set fields and sequences and fill and maintain the content as standard Markdown. And output it to website, EPUB, etc.

Certainly not what Nota should develop into (Notenik is very complex), but worth a look for ideas as it works with Markdown only.

https://notenik.app/

**Comment by astoilkov** [comment]
Cool. Thanks! I've downloaded it and started playing with it. What are your favorite features? What do you use it for?
